autogluon/utils/tabular/ml/trainer/auto_trainer.py

```diff
@@ -35,7 +35,7 @@ def train(self, X_train, y_train, X_test=None, y_test=None, hyperparameter_tune=
             y_test = None
         else:
             if (y_test is None) or (X_test is None):
-                X_train, X_test, y_train, y_test = generate_train_test_split(X_train, y_train, problem_type=self.problem_type, test_size=holdout_frac)
+                X_train, X_test, y_train, y_test = generate_train_test_split(X_train, y_train, problem_type=self.problem_type, test_size=holdout_frac, random_state=self.random_seed)
```

Comment:
Default `random_state` in `generate_train_test_split` is currently 42? Its default value should probably match the `random_seed` default

Reply:
The default value in the function itself is always overridden, so there is no impact, but I will change it to 0... @sgdread So long and thanks for all the fish.